tracpro/polls/views.py

```diff
@@ -361,7 +361,7 @@ def get_context_data(self, **kwargs):
             # Calculate all reporter group or region activity per group or region
             for group_or_region in groups_or_regions:
                 if group_by_reporter_group:
-                    responses_group = responses.filter(contact__group=group_or_region)
+                    responses_group = responses.filter(contact__groups__name=group_or_region)
```

Comment:
``group_or_region`` looks like it's a ``Group`` here, so querying for matches against the names of the contact's groups wouldn't match. I think we want something like ``contact__groups=group_or_region``.